caluma/core/filters.py

```diff
@@ -489,6 +489,59 @@ def merge_querysets(cls, default_queryset, queryset):
         queryset.query.select_related = default_queryset.query.select_related
         return queryset
 
+    @classmethod
+    def connection_resolver(
+        cls,
+        resolver,
+        connection,
+        default_manager,
+        max_limit,
+        enforce_first_or_last,
+        filterset_class,
+        filtering_args,
+        root,
+        info,
+        **args,
+    ):
+        class QuerysetManager(object):
```

Comment:
This builds a new class for each request.. If we want to have a custom manager optimized for our use case, why not move it to the model itself, or at least out to module scope? That seems to be the better place for it

Reply:
this class uses outscope so don't quite see how this could be done on model level?

Reply:
updated so it actually calls super to skip filtering.